Sources/OpenTelemetryApi/Baggage/BaggageBuilder.swift

```diff
@@ -43,6 +43,6 @@ public protocol BaggageBuilder: AnyObject {
     /// - Parameter key: the EntryKey which will be removed.
     @discardableResult func remove(key: EntryKey) -> Self
 
-    /// Creates a Baggage from this builder.
-    func build() -> Baggage
+    /// Creates a Baggage from this builder or nil if it has no entries itself or the parent
```

Comment:
please rephrase. This sentence is hard to understand

Reply:
Yes, you are true. I will remove the change,it was trying to explain the implementation of the `DefaultBaggageBuilder`, and this is the protocol. Just made no sense